templates/.github/CONTRIBUTING.md

```diff
@@ -52,7 +52,7 @@ but, basically, it all boils down to the following:
     ```haskell
     {- SPDX-FileCopyrightText: 2019 Serokell <https://serokell.io>
      -
-     - SPDX-License-Identifier: MPL-2.0
+     - SPDX-License-Identifier: LicenseRef-ReplaceMe
```

Comment:
I would preserve `MPL-2.0` here because it's our default choice.

Reply:
Well, I mean, this way it is guaranteed to make the person think.
(The real reason I did this was that the lint tool saw these lines, thought it was a header, and reported that it doesn’t know the MPL-2.0 license :).